build.gradle.kts

```diff
@@ -53,6 +53,7 @@ tasks {
   register("updateVersionInDocs") {
     group = "documentation"
     doLast {
+      val version = findProperty("release.version")
```

Comment:
The release doc calls to run `./gradlew updateVersionInDocs -Prelease.version=x.y.z`, which doesn't work if reading the version from `version.gradle.kts`.